sa-gateway/k8s/api-gateway-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
shouldn't we have just 1 deployment that will deploy all our services together?

Reply:
I'm not sure whether it is possible. These scripts were generated automatically and I left them as they were. If you need everything in a one single script, see `compiled.yaml`